guides/source/active_record_callbacks.md

```diff
@@ -3,108 +3,137 @@
 Active Record Callbacks
 =======================
 
-This guide teaches you how to hook into the life cycle of your Active Record objects.
+This guide teaches you how to hook into the life cycle of your Active Record
+objects.
 
 After reading this guide, you will know:
 
-* When certain events occur during the life of an Active Record object
-* How to create callback methods that respond to events in the object life cycle.
-* How to create special classes that encapsulate common behavior for your callbacks.
+* When certain events occur during the life of an Active Record object.
+* How to register, run, and skip callbacks that respond to these events.
+* How to create relational, association, conditional, and transactional
+  callbacks.
+* How to create special classes that encapsulate common behavior for your
+  callbacks to be reused.
 
 --------------------------------------------------------------------------------
 
 The Object Life Cycle
 ---------------------
 
-During the normal operation of a Rails application, objects may be created, updated, and destroyed. Active Record provides hooks into this *object life cycle* so that you can control your application and its data.
+During the normal operation of a Rails application, objects may be [created,
+updated, and
+destroyed](active_record_basics.html#crud-reading-and-writing-data). Active
+Record provides hooks into this object life cycle so that you can control your
+application and its data.
 
-Callbacks allow you to trigger logic before or after an alteration of an object's state.
+Callbacks allow you to trigger logic before or after a change to an
+object's state. They are methods that get called at certain moments of an
+object's life cycle. With callbacks it is possible to write code that will run
+whenever an Active Record object is initialized, created, saved, updated, deleted, validated,
+or loaded from the database.
 
 ```ruby
 class Baby < ApplicationRecord
-  after_create -> { puts "Congratulations!" }
+  after_create -> { Rails.logger.info("Congratulations!") }
```

Comment:
ActiveRecord models do have logging facilities, so maybe there is no need to be explicit with `Rails.logger...` and use local logger instead?

```ruby
after_create -> { logger.info("Congratulations!") }
```

Applies to (probably) all other `puts` -> `Rails.logger.info` changes bellow.

Reply:
@tadas-s, thanks for bringing this up!  However, I think it may be preferable to maintain the explicitness to ensure consistency thus reducing the potential confusion for new developers who are still familiarizing themselves with the basics.